tests/FSharp.Compiler.ComponentTests/CompilerOptions/fsc/crossoptimize.fs

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+namespace FSharp.Compiler.ComponentTests.CompilerOptions
+
+open Xunit
+open FSharp.Test
+open FSharp.Test.Compiler
+
+//# Sanity check - simply check that the option is valid
```

Comment:
Maybe we could have just one test for this with `[<Theory>]` and put all the flags in `InlineData`? It wouldn't have to be just for crossoptimize, but whatever flags we want sanity check for...

Reply:
Sure, i can change that tomorrow.